rhel8/templates/csv/services_enabled.csv

```diff
@@ -1,10 +1,14 @@
-auditd,audit,
-chronyd,chrony,
+auditd,,
+chronyd_or_ntpd,chrony,
```

Comment:
Comment in https://github.com/ComplianceAsCode/content/pull/4739/files#diff-6359728e005606555ad968d6eb8d980bR4 says it needs to be static.

Is it still the case?

Reply:
I see now - there is no need for template here. The rule name is `service_chronyd_or_ntpd_enabled`, so it can't have a template, as templates require the rule ID to contain the service name.

Reply:
Definition for `service_chronyd_or_ntpd_enabled` still depends on the service checks for `chronyd`and `ntpd` though.

Reply:
I see, it is about creating OVALs for extended definitions. Fixed by rebase.